bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/BigtableRetriesExhaustedException.java

```diff
@@ -21,6 +21,8 @@
  * An Exception that is thrown when an operation fails, even in the face of retrues.
  */
 public class BigtableRetriesExhaustedException extends IOException {
+  private static final long serialVersionUID = 6905598607595217072L;
```

Comment:
All Java exceptions are Serializable.  The JVM spits out warnings if serialVersionUID isn't there.